### DISCUSSION TITLE
koyeb后端刮不出来了

### DISCUSSION THREAD

**Discussion by fatbearlee** [metatube-community/jellyfin-plugin-metatube#301] [General]
突然就无法刮梢了，然后打开koyeb的地址显示
upstream connect error or disconnect/reset before headers. retried and the latest reset reason: connection termination
尝试重新部署依然是这个提示。这有什么解决办法么？

**Comment by Chia-0918** [comment]
我用公開的也不能用了

**Comment by fatbearlee** [comment]
之前的koyeb旧账号无论是直接在已部署的Redeploy还是删除已部署的重新部署都无效一直提示“upstream connect error or disconnect/reset before headers. retried and the latest reset reason: connection termination”，不清楚是什么情况。
然后我我重新注册了个koyeb的账号，然后重新部署了一次就可以了。

**Comment by poko11** [comment]
自建一个，最重要的是要在计划任务里更新插件

**Comment by shidonsiu** [comment]
同问，部署好了都刮捎不出

**Reply by zhouyan553** [reply]
同样，使用付费日本节点也刮不出来，其余后端需要企业邮箱，没条件
